src/site/content/sitemap.xml.njk

```diff
@@ -6,8 +6,11 @@ permalink: "/sitemap.xml"
 <urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
 {%- for page in collections.all %}
 {%- if not page.data.noindex and not page.data.draft %}
+  {# Verify that the page has a permalink so we don't pull in things like #}
+  {# the service worker partial. #}
+  {# But use the page's fileSlug so we can strip out the language directory. #}
   {%- if page.url %}
-  {% set absoluteUrl %}{{ page.url | absoluteUrl(site.url) }}{% endset %}
+  {% set absoluteUrl %}{{ page.data.canonicalUrl }}{% endset %}
```

Comment:
I'm not entirely clear why would we want to strip language from url in a sitemap? We want e.g. google bot to see both /en/blah and /pl/blah, right?

Reply:
I think we'd only want to remove english, other languages should have their prefix. @robdodson?

Reply:
I think Michael is correct, though I'm not an SEO expert.

But I think the idea is we'd want google bot to index:

/web-vitals/
/pl/web-vitals/
/ja/web-vitals/